### PR TITLE
README: use GPS flatpak id

### PR DESCRIPTION
**Pull request by dabrain34**
### Please confirm your submission meets all the criteria

- [x ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ x] My pull request follows the instructions at [App Submission][submission].
- [ x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ flatpak install flathub org.gnome.Recipes
 To install applications from the beta branch, use the following:
```
Comment (bbhtt) [inline README.md]: There's no `beta` branch here

Comment (dabrain34) [reply README.md]: true

Comment (dabrain34) [reply README.md]: in the future maybe ...

**Comment by bbhtt** [comment]
These files are actually leftovers from the flathub repo. You should delete the .github directory, and contributing,copying file. Keep the readme if you want to fill in some info.

**Comment by dabrain34** [comment]
> These files are actually leftovers from the flathub repo. You should delete the .github directory, and contributing,copying file. Keep the readme if you want to fill in some info.

Ok to remove the ".github" and  but are you sure we should remove the contributing and COPYING ?

**Comment by bbhtt** [comment]
Yea, those come from the master branch here https://github.com/flathub/flathub while application submissions go to the `new-pr` branch which is empty. 

My guess is this app was submitted targeting the wrong branch - `master` and thus inherited the leftovers.

**Comment by dabrain34** [comment]
should be okay now @bbhtt

**Pull request merged**